admin_manual/installation/source_installation.rst

```diff
@@ -15,6 +15,8 @@ archive <https://nextcloud.com/install/>`_.
 .. note:: Admins of SELinux-enabled distributions such as CentOS, Fedora, and
    Red Hat Enterprise Linux may need to set new rules to enable installing
    Nextcloud. See :ref:`selinux_tips_label` for a suggested configuration.
+   
+You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server. It will setup everything for you and include scripts for automated installtion of apps like; Collabora, OnlyOffice, Talk and so on.
```

Comment:
```suggestion
You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server. It will setup everything for you and include scripts for automated instalaltion of apps like; Collabora, OnlyOffice, Talk and so on.
```